packages/amesos2/src/Amesos2_Util.hpp

```diff
@@ -66,11 +66,15 @@
 
 #include "Amesos2_TypeDecl.hpp"
 #include "Amesos2_Meta.hpp"
+#include "Amesos2_Kokkos_View_Copy_Assign.hpp"
 
 #ifdef HAVE_AMESOS2_EPETRA
 #include <Epetra_Map.h>
 #endif
 
+#ifdef HAVE_AMESOS2_METIS
+#include "metis.h" // to discuss, remove from header?
```

Comment:
Please do -- including TPL headers in Trilinos headers risks breaking applications' builds (e.g., if they use extern declarations instead of headers, or if the versions don't quite line up).

Reply:
Discussed plan was to have this use zoltan2 ordering instead of Metis.h. Will need to discuss best way to make that work with the new kokkos adapter.

For the cuSolver headers, those are also in the include stack, via Amesos2_cuSOLVER_FunctionMap.hpp which is currently accessed directly by the decl file Amesos2_cuSOLVER_decl.hpp. The other Amesos2 solvers also include this way for TPLs so the plan is to not make any changes for that right now.